io/transmat/gs/gs_token.go

```diff
@@ -0,0 +1,71 @@
+package gs
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/storage/v1"
+)
+
+/*
+ Environment variable used to directly pass an OAuth2 access token to repeatr
+*/
+const EnvGsAccessToken = "GS_ACCESS_TOKEN"
+
+/*
+ Environment variable used to pass the path of
+ a Google Developers service account JSON key file
+ which is then used for two-legged OAuth to create an Access Token
+*/
+const EnvGsAccountFile = "GS_SERVICE_ACCOUNT_FILE"
```

Comment:
woof :dog2: 
Which environment variables to use and in what contexts can become quite complex. If people download private keys instead of the JSON file then we could load the account name and private keys separately. We could load refresh tokens or we could implement some portion of 3-legged oauth that helps people create tokens using other methods. All of this is extra complexity that we don't need at the moment. I'd like to see some use cases before going full bore into features. And there's a few `TODO`s in there that could be a lot more interesting/useful to knock down such as composite objects.

I agree on avoiding poking extra externals. We aren't trying to support GCE and it should not be wrapped into default behavior.